### PR TITLE
[FLINK-27120][docs-zh] Translate "Gradle" tab of development > conf > overview

### PR DESCRIPTION
**Pull request by zoltar9264**
## What is the purpose of the change

[FLINK-27120](https://issues.apache.org/jira/browse/FLINK-27120)  "Gradle" tab of "Application Development > Project Configuration > Overview" to Chinese.

related [PR](https://github.com/apache/flink/pull/18609).

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ $ curl https://flink.apache.org/q/quickstart.sh | bash -s {{< version >}}
 
 {{< /tab >}}
 {{< tab "Gradle" >}}
-您可以使用如下的 Gradle 构建脚本或快速启动脚本创建一个项目。
+您可以创建一个空项目，您需要在其中手动创建 `src/main/java` 和 `src/main/resources` 目录并开始在其中编写一些类，使用如下 Gradle 构建脚本或下面提供的快速启动脚本以获得功能齐全的启动项目。
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:70]: 您 -> 你

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ $ curl https://flink.apache.org/q/quickstart.sh | bash -s {{< version >}}
...
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:71]: missing link tag ```<a name="xxx"></a>```

**Review by RocMarshal — CHANGES_REQUESTED** [review]

File: docs/content.zh/docs/dev/configuration/overview.md (hunk 2 above)
Comment (zoltar9264) [reply docs/content.zh/docs/dev/configuration/overview.md:71]: Thanks for your comment, I had add link tag  <a name="gradle-build-script"></a> and <a name="quickstart-script"></a> , but it seems not work inner {{ tab }}. Is that a another problem.

By the way, the other part of this page also use “您” and missing link tag , should I add hotfix commit to fix that ?

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 3]
```diff
@@ -85,25 +86,31 @@ plugins {
...
 tasks.withType(JavaCompile) {
-	options.encoding = 'UTF-8'
+    options.encoding = 'UTF-8'
 }
 applicationDefaultJvmArgs = ["-Dlog4j.configurationFile=log4j2.properties"]
 
 // declare where to find the dependencies of your project
 repositories {
     mavenCentral()
+    maven {
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:108]: Minor confusion from me, Was the new lines caused by the English edition page ?

**Review by RocMarshal — COMMENTED** [review]
Thanks @zoltar9264  for the update.  Looks better and I left some comments. PTAL.

File: docs/content.zh/docs/dev/configuration/overview.md (hunk 3 above)
Comment (zoltar9264) [reply docs/content.zh/docs/dev/configuration/overview.md:108]: Yes, most code modification is copy from English edition page.

And  as previous reply mentioned , the other part of this page also use “您” and missing link tag , should I add hotfix commit to fix that ?

Comment (RocMarshal) [reply docs/content.zh/docs/dev/configuration/overview.md:108]: @zoltar9264 IMO, you could fix it together in the current pr. 
Feel free go ahead

Comment (zoltar9264) [reply docs/content.zh/docs/dev/configuration/overview.md:112]: Thanks @RocMarshal , I had fix it together.

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 4]
```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+<a name="project-configuration"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:39]:
```suggestion
<a name="project-configuration"></a>

```

Comment (RocMarshal) [reply docs/content.zh/docs/dev/configuration/overview.md:39]: It's recommended that There should be an empty line between tag and the section title.

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 5]
```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
...
 # 项目配置
 
-本节将向您展示如何通过流行的构建工具 ([Maven]({{< ref "docs/dev/configuration/maven" >}})、[Gradle]({{< ref "docs/dev/configuration/gradle" >}})) 配置您的项目，必要的依赖项（比如[连接器和格式]({{< ref "docs/dev/configuration/connector" >}})），以及覆盖一些[高级]({{< ref "docs/dev/configuration/advanced" >}})配置主题。
+本节将向你展示如何通过流行的构建工具 ([Maven]({{< ref "docs/dev/configuration/maven" >}})、[Gradle]({{< ref "docs/dev/configuration/gradle" >}})) 配置你的项目，必要的依赖项（比如[连接器和格式]({{< ref "docs/dev/configuration/connector" >}})），以及覆盖一些[高级]({{< ref "docs/dev/configuration/advanced" >}})配置主题。
 
 每个 Flink 应用程序都依赖于一组 Flink 库。应用程序至少依赖于 Flink API，此外还依赖于某些连接器库（比如 Kafka、Cassandra），以及用户开发的自定义的数据处理逻辑所需要的第三方依赖项。
 
+<a name="getting-started"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:46]:
```suggestion
<a name="getting-started"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 6]
```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
...
 
 要开始使用 Flink 应用程序，请使用以下命令、脚本和模板来创建 Flink 项目。
 
 {{< tabs "creating project" >}}
 {{< tab "Maven" >}}
 
-您可以使用如下的 Maven 命令或快速启动脚本，基于[原型](https://maven.apache.org/guides/introduction/introduction-to-archetypes.html)创建一个项目。
+你可以使用如下的 Maven 命令或快速启动脚本，基于[原型](https://maven.apache.org/guides/introduction/introduction-to-archetypes.html)创建一个项目。
 
+<a name="maven-command"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:56]:
```suggestion
<a name="maven-command"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 7]
```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
...
 ```bash
 $ mvn archetype:generate                \
   -DarchetypeGroupId=org.apache.flink   \
   -DarchetypeArtifactId=flink-quickstart-java \
   -DarchetypeVersion={{< version >}}
 ```
-这允许您命名新建的项目，而且会交互式地询问 groupId、artifactId、package 的名字。
+这允许你命名新建的项目，而且会交互式地询问 groupId、artifactId、package 的名字。
 
+<a name="quickstart-script"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:66]:
```suggestion
<a name="quickstart-script"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 8]
```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
...
 ```bash
 $ curl https://flink.apache.org/q/quickstart.sh | bash -s {{< version >}}
 ```
 
 {{< /tab >}}
 {{< tab "Gradle" >}}
-您可以使用如下的 Gradle 构建脚本或快速启动脚本创建一个项目。
+你可以创建一个空项目，你需要在其中手动创建 `src/main/java` 和 `src/main/resources` 目录并开始在其中编写一些类，使用如下 Gradle 构建脚本或下面提供的快速启动脚本以获得功能齐全的启动项目。
 
+<a name="gradle-build-script"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:76]:
```suggestion
<a name="gradle-build-script"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 9]
```diff
@@ -158,6 +175,7 @@ shadowJar {
 rootProject.name = 'quickstart'
 ```
 
+<a name="quickstart-script"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:178]:
```suggestion
<a name="quickstart-script"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 10]
```diff
@@ -166,21 +184,23 @@ bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version
 {{< /tab >}}
 {{< /tabs >}}
 
+<a name="which-dependencies-do-you-need"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:187]:
```suggestion
<a name="which-dependencies-do-you-need"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 11]
```diff
@@ -166,21 +184,23 @@ bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version
...
 ## 需要哪些依赖项？
 
-要开始一个 Flink 作业，您通常需要如下依赖项：
+要开始一个 Flink 作业，你通常需要如下依赖项：
 
-* Flink API, 用来开发您的作业
-* [连接器和格式]({{< ref "docs/dev/configuration/connector" >}}), 以将您的作业与外部系统集成
-* [测试实用程序]({{< ref "docs/dev/configuration/testing" >}}), 以测试您的作业
+* Flink API, 用来开发你的作业
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:192]:
```suggestion
* Flink API，用来开发你的作业
```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 12]
```diff
@@ -166,21 +184,23 @@ bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version
...
+* [连接器和格式]({{< ref "docs/dev/configuration/connector" >}}), 以将你的作业与外部系统集成
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:193]:
```suggestion
* [连接器和格式]({{< ref "docs/dev/configuration/connector" >}})，以将你的作业与外部系统集成
```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 13]
```diff
@@ -166,21 +184,23 @@ bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version
...
+* [测试实用程序]({{< ref "docs/dev/configuration/testing" >}}), 以测试你的作业
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:194]:
```suggestion
* [测试实用程序]({{< ref "docs/dev/configuration/testing" >}})，以测试你的作业
```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 14]
```diff
@@ -166,21 +184,23 @@ bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version
...
 
-除此之外，若要开发自定义功能，您还要添加必要的第三方依赖项。
+除此之外，若要开发自定义功能，你还要添加必要的第三方依赖项。
 
+<a name="flink-apis"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:198]:
```suggestion
<a name="flink-apis"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 15]
```diff
@@ -189,18 +209,20 @@ Flink提供了两大 API：[Datastream API]({{< ref "docs/dev/datastream/overvie
 | [Table API + DataStream]({{< ref "docs/dev/table/data_stream_api" >}})            | `flink-table-api-java-bridge`                       |
 | [Table API + DataStream Scala 版]({{< ref "docs/dev/table/data_stream_api" >}})   | `flink-table-api-scala-bridge{{< scala_version >}}` |
 
-您只需将它们包含在您的构建工具脚本/描述符中，就可以开发您的作业了！
+你只需将它们包含在你的构建工具脚本/描述符中，就可以开发你的作业了！
 
+<a name="running-and-packaging"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:214]:
```suggestion
<a name="running-and-packaging"></a>

```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 16]
```diff
@@ -189,18 +209,20 @@ Flink提供了两大 API：[Datastream API]({{< ref "docs/dev/datastream/overvie
...
 
-根据经验，我们**建议**将应用程序代码及其所有必需的依赖项打包进一个 fat/uber JAR 中。这包括打包您作业用到的连接器、格式和第三方依赖项。此规则**不适用于** Java API、DataStream Scala API 以及前面提到的运行时模块，它们已经由 Flink 本身提供，**不应**包含在作业的 uber JAR 中。您可以把该作业 JAR 提交到已经运行的 Flink 集群，也可以轻松将其添加到 Flink 应用程序容器镜像中，而无需修改发行版。
+根据经验，我们**建议**将应用程序代码及其所有必需的依赖项打包进一个 fat/uber JAR 中。这包括打包你作业用到的连接器、格式和第三方依赖项。此规则**不适用于** Java API、DataStream Scala API 以及前面提到的运行时模块，它们已经由 Flink 本身提供，**不应**包含在作业的 uber JAR 中。你可以把该作业 JAR 提交到已经运行的 Flink 集群，也可以轻松将其添加到 Flink 应用程序容器镜像中，而无需修改发行版。
 
+<a name="whats-next"></a>
 ## 下一步是什么？
 
-* 要开发您的作业，请查阅 [DataStream API]({{< ref "docs/dev/datastream/overview" >}}) 和 [Table API & SQL]({{< ref "docs/dev/table/overview" >}});
-* 关于如何使用特定的构建工具打包您的作业的更多细节，请查阅如下指南：
+* 要开发你的作业，请查阅 [DataStream API]({{< ref "docs/dev/datastream/overview" >}}) 和 [Table API & SQL]({{< ref "docs/dev/table/overview" >}});
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:224]:
```suggestion
* 要开发你的作业，请查阅 [DataStream API]({{< ref "docs/dev/datastream/overview" >}}) 和 [Table API & SQL]({{< ref "docs/dev/table/overview" >}})；
```

File: docs/content.zh/docs/dev/configuration/overview.md [hunk 17]
```diff
@@ -189,18 +209,20 @@ Flink提供了两大 API：[Datastream API]({{< ref "docs/dev/datastream/overvie
...
+<a name="running-and-packaging"></a>
 ## 运行和打包
 
-如果您想通过简单地执行主类来运行你的作业，您需要 classpath 里有 `flink-runtime`。对于 Table API 程序，您还需要 `flink-table-runtime` 和 `flink-table-planner-loader`。
+如果你想通过简单地执行主类来运行你的作业，你需要 classpath 里有 `flink-runtime`。对于 Table API 程序，你还需要 `flink-table-runtime` 和 `flink-table-planner-loader`。
 
-根据经验，我们**建议**将应用程序代码及其所有必需的依赖项打包进一个 fat/uber JAR 中。这包括打包您作业用到的连接器、格式和第三方依赖项。此规则**不适用于** Java API、DataStream Scala API 以及前面提到的运行时模块，它们已经由 Flink 本身提供，**不应**包含在作业的 uber JAR 中。您可以把该作业 JAR 提交到已经运行的 Flink 集群，也可以轻松将其添加到 Flink 应用程序容器镜像中，而无需修改发行版。
+根据经验，我们**建议**将应用程序代码及其所有必需的依赖项打包进一个 fat/uber JAR 中。这包括打包你作业用到的连接器、格式和第三方依赖项。此规则**不适用于** Java API、DataStream Scala API 以及前面提到的运行时模块，它们已经由 Flink 本身提供，**不应**包含在作业的 uber JAR 中。你可以把该作业 JAR 提交到已经运行的 Flink 集群，也可以轻松将其添加到 Flink 应用程序容器镜像中，而无需修改发行版。
 
+<a name="whats-next"></a>
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/configuration/overview.md:221]:
```suggestion
<a name="whats-next"></a>

```

**Comment by zoltar9264** [comment]
Thanks for your guide @RocMarshal ! I hive committed all of your suggestion.

**Comment by RocMarshal** [comment]
> Thanks for your guide @RocMarshal ! I hive committed all of your suggestion. Need I squash those commit into one ?

Cool~ @zoltar9264 .
Please let me have a final check before your squashing commits.

**Review by RocMarshal — APPROVED** [review]
@zoltar9264 LGTM +1 (non-binding) once CI passed.

now ping @MartijnVisser @wuchong .

**Comment by zoltar9264** [comment]
> @zoltar9264 LGTM +1 (non-binding) once CI passed.
> 
> now ping @MartijnVisser @wuchong .

kindly ping @MartijnVisser @wuchong

**Comment by MartijnVisser** [comment]
@zoltar9264 Thanks, let me squash commits and rebase to merge it. I'll take care. @RocMarshal Thanks for the review

**Review by MartijnVisser — APPROVED** [review]
LGTM

**Pull request merged**